.github/workflows/lint-and-test.yaml

```diff
@@ -0,0 +1,85 @@
+name: Lint and test charts
+on:
+  push:
+
+  # Workflow dispatch listener to enable on-demand acceptance test runs on external PRs.
+  # How to use this:
+  # * Do a sanity check on the submitted PR
+  # * Copy the most recent commit hash of the PR branch
+  # * Go to 'Actions' -> 'Run acceptance tests' -> 'Run workflow'
+  # * Fill in the following:
+  #   * `checkout-repo`: `<PR author>/op-scim-helm`
+  #   * `checkout-ref`: <copied commit hash>
+  #   * `branch`: `acceptance-tests-on-forks`
+  workflow_dispatch:
+    inputs:
+      checkout-repo:
+        description: "Checkout repo"
+        required: false
+      checkout-ref:
+        description: "Chekout ref"
+        required: false
+
+jobs:
+  lint-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          repository: ${{ github.event.inputs.checkout-repo }}
+          ref: ${{ github.event.inputs.checkout-ref }}
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.1
+
+      # Set up Python for ct lint, which needs Python for Yamale and yamllint
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up ct CLI
+        uses: helm/chart-testing-action@v2.1.0
+        with:
+          version: v3.4.0
+
+      - name: Lint chart
+        run: ct lint --config ct.yaml
+
+      - name: Check if chart has changed
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --config ct.yaml)
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
```

Comment:
Can you explain this line in a bit more detail? This seems to be a logging line to my eye. Or is it being written into the `ct.yaml` file? What are the `::` for?

Reply:
Good catch. Sorry for not adding a comment calling it out. This is actually part of GitHub actions, specifically [workflow commands to access toolkit functions](https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#using-workflow-commands-to-access-toolkit-functions).

We set the value and then opt to skip the next step in the workflow depending on the value:
<img width="604" alt="image" src="https://user-images.githubusercontent.com/11049609/146999345-bdb372cd-d5ec-4b1b-ab1b-a411231e44dd.png">

Reply:
Ah I understand. I think that line would indeed benefit from a comment.

Reply:
Good idea.

- [ ] add comment to explain